cdc/scheduler/internal/v3/replication/replication_manager.go

```diff
@@ -625,9 +626,12 @@ func (r *Manager) AdvanceCheckpoint(
 				}
 
 				// Find the minimum checkpoint ts and resolved ts.
-				if watermark.CheckpointTs > table.Checkpoint.CheckpointTs {
+				if watermark.CheckpointTs > table.Checkpoint.CheckpointTs ||
+					(watermark.CheckpointTs == table.Checkpoint.CheckpointTs &&
```

Comment:
Please add some comment why we need to check `watermark.CheckpointTs == table.Checkpoint.CheckpointTs`?